pkg/controller/cloud/pvlcontroller.go

```diff
@@ -286,38 +282,13 @@ func (pvlc *PersistentVolumeLabelController) updateVolume(vol *v1.PersistentVolu
 	return nil
 }
 
-func removeInitializer(initializers *metav1.Initializers, name string) *metav1.Initializers {
-	if initializers == nil {
-		return nil
-	}
-
-	var updated []metav1.Initializer
-	for _, pending := range initializers.Pending {
-		if pending.Name != name {
-			updated = append(updated, pending)
-		}
-	}
-	if len(updated) == len(initializers.Pending) {
-		return initializers
-	}
-	if len(updated) == 0 {
-		return nil
-	}
-
-	return &metav1.Initializers{Pending: updated}
+func markInitialized(vol *v1.PersistentVolume) {
+	// TODO: mark initialized using a different field, since initializers are not being promoted past alpha, or convert to an admission plugin
 }
 
 // needsInitialization checks whether or not the PVL is the next pending initializer.
-func needsInitialization(initializers *metav1.Initializers, name string) bool {
-	if initializers == nil {
-		return false
-	}
-
-	if len(initializers.Pending) == 0 {
-		return false
-	}
-
-	// There is at least one initializer still pending so check to
-	// see if the PVL is the next in line.
-	return initializers.Pending[0].Name == name
+func needsInitialization(vol *v1.PersistentVolume) bool {
+	// TODO: determine whether initialization is required based on a different attribute,
```

Comment:
Just a note for reference: this should ideally be done for v1.14 (in a future PR though)